fe/fe-core/src/main/java/org/apache/doris/nereids/parser/LogicalPlanBuilder.java

```diff
@@ -6116,14 +6116,15 @@ public LogicalPlan visitShowConvertLsc(ShowConvertLscContext ctx) {
 
     @Override
     public LogicalPlan visitKillQuery(KillQueryContext ctx) {
```

Comment:
based on old planner, if specify QUERY KEYWORD, it always means to kill query, and the query id is whatever INTEGER_VALUE or STRING_LITERAL. Looks like the behavior is changed in this pr?
```
kill_stmt ::=
    KW_KILL INTEGER_LITERAL:value
    {:
        RESULT = new KillStmt(true, value.intValue());
    :}
    | KW_KILL KW_CONNECTION INTEGER_LITERAL:value
    {:
        RESULT = new KillStmt(true, value.intValue());
    :}
    | KW_KILL KW_QUERY INTEGER_LITERAL:value
    {:
        RESULT = new KillStmt(false, value.intValue());
    :}
    | KW_KILL KW_QUERY STRING_LITERAL:value
    {:
        RESULT = new KillStmt(value);
    :}
    ;
```

Reply:
No, there are 2 ways to kill query:
1. Kill query by query id: `KILL QUERY "query_id";`
2. Kill query by connection id: `KILL QUERY connection_id`, where connection_id is an integer, not a string.

Reply:
> No, there are 2 ways to kill query:
> 
> 1. Kill query by query id: `KILL QUERY "query_id";`
> 2. Kill query by connection id: `KILL QUERY connection_id`, where connection_id is an integer, not a string.

the grammer is very weird. `KILL QUERY` but need a params of connection_id. could lead to misunderstand for people not read doc very carefully.

Reply:
> > No, there are 2 ways to kill query:
> > 
> > 1. Kill query by query id: `KILL QUERY "query_id";`
> > 2. Kill query by connection id: `KILL QUERY connection_id`, where connection_id is an integer, not a string.
> 
> the grammer is very weird. `KILL QUERY` but need a params of connection_id. could lead to misunderstand for people not read doc very carefully.

This is compatible with [MySQL Kill Statement](https://dev.mysql.com/doc/refman/8.4/en/kill.html)